CommonLoggingFody/NugetAssets/Anotar.CommonLogging.Fody.nuspec

```diff
@@ -16,7 +16,7 @@
     <tags>Logging, ILWeaving, Fody, Cecil, Common.Logging</tags>
     <dependencies>
       <dependency id="Fody" version="1.29.2"/>
-      <dependency id="Common.Logging.Core" version="3.2.0"/>		
+      <dependency id="Common.Logging.Portable" version="3.3.0"/>
```

Comment:
this wont work since depending on the use case people may either reference https://www.nuget.org/packages/Common.Logging.Portable/ or  https://www.nuget.org/packages/Common.Logging/

you cant force a choice on the user